packages/react-native-bridge/android/publish-aztec-and-bridge.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# 1. Verify the version argument is passed. We use the same version for react-native-aztec and react-native-bridge libraries.
+VERSION=$1
+if [[ -z $VERSION ]]; then
+    echo "This script requires the Bintray version to be passed as an argument."
+    echo "Example usage: './publish-aztec-and-bridge.sh \$VERSION'"
+    exit 1
+fi
+
+# 2. Check if we can call gradlew successfully
+./gradlew --help > /dev/null 2>&1
+gradlew_exists=$?
+if [ $gradlew_exists -ne 0 ]; then
+    echo "Gradle wrapper is not available. Make sure to run this script from '/packages/react-native-bridge/android'"
+	exit 1
+fi
+
+# 3. Clean and copy the bundle
+./gradlew clean -q
+mkdir -p react-native-bridge/build/assets
+cp ../../../../bundle/android/App.js ./react-native-bridge/build/assets/index.android.bundle
+
+if [ $? -ne 0 ]; then
+	echo "Make sure to run 'npm install' & 'npm run bundle:android' from the root folder of the project before running this script."
```

Comment:
What do you think of using this in the output so users that get this error can copy-paste the command and run it in one go?

```suggestion
	echo "Make sure to run `npm install && npm run bundle:android` from the root folder of the project before running this script."
```

Reply:
Addressed in https://github.com/oguzkocer/gutenberg/commit/4c67a6c540ba2fda7776c9e3b93d7f8381bf80d4